lib/vanagon/platform/windows.rb

```diff
@@ -67,7 +67,39 @@ def generate_nuget_package(project)
         "cp $(tempdir)/#{project.name}/#{project.name}-#{@architecture}.#{nuget_package_version(project.version, project.release)}.nupkg ./output/#{target_dir}/#{nuget_package_name(project)}"]
       end
 
-      # Method to derive the package name for the project.
+      # The specific bits used to generate a windows msi package for a given project
+      # [TBD - More to add]
+      #
+      # @param project [Vanagon::Project] project to build a msi package of
+      # @return [Array] list of commands required to build an msi package for the given project from a tarball
+      def generate_msi_package(project)
```

Comment:
So, what is this going to look like? I'm assuming this is where we're going to call harvest, heat, light, etc? What is the order that's going to happen in? Do we know what WXS files we have to copy over, and where we have to copy them to get the commands to run correctly? Do we just have to unpack the tarball into the stagedir, or somewhere else?